.github/workflows/test-salt-linux.yml

```
@@ -39,7 +39,8 @@ jobs:
       image: >-
         ${{ fromJson('{
           7: "quay.io/centos/centos:centos7",
-          8: "quay.io/centos/centos:stream8"
+          8: "hub.docker.com/redhat:ubi8",
+          9: "hub.docker.com/redhat:ubi9"
```

based on the error from the ash-linux test, i think the registry needs to be `docker.io`...

```suggestion
          8: "docker.io/redhat:ubi8",
          9: "docker.io/redhat:ubi9"
```

---

out-of-date: UBIs are a dead-end (they can be _made_ to work, but it's significantly-higher effort – both to get working and, likely, to _keep_ working – than using either Rocky or Alma)